Vaishnavi/ass1.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1><b>Merlo J. Pusey</b></h1>
+    <hr/>
+<i> From Wikipedia, the free encyclopedia</i>
+<br/>
+        <img src=" https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg"/>
+       <p><strong>Merlo John Pusey</strong> (February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington, D.C.)<sup>[1][2] </sup>was an <ins>American biographer</ins> and <ins>editorial writer</ins>. He won the 1952 Pulitzer Prize for Biography or Autobiography
+        and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.</p>
+        
+       <p> Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college newspaper. He later became
+        a reporter and assistant city editor at The Deseret News in Salt Lake City.<sup>[3]</sup></p>
+        
+        <p><del>Pussy</del> Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946, continuing to contribute occasional pieces until about two years before his death.</p>
+        <p align = "left"> <ol>
+        <h2 align="left">Contents</h2>
+       
+            <li><a href="">
+                    Books</li>
+            </a>
+        <li><a href="">
+            References</a>
+            </li>
+    </ol>
+    </p>
+        
+       <h2>Books</h2> 
+       <hr/>
+       <ul>
+        <li>The Supreme Court Crisis<i>(Macmillan, 1937)</i></li>
+        <li>Big Government: Can We Control It? <i>(Harper, 1945)</i></li>
+       <li> Charles Evans Hughes<i>(2 vols., Macmillan, 1951)</i> </li>
+       <li>Eisenhower, the President <i>(Macmillan, 1956)</i></li> 
+       </ul>
+        <h2>References</h2>
+        <hr/>
+        <ul>
+            <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286. Retrieved 2016-03-24.</li>
+               <li> Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York Times. ISSN 0362-4331. Retrieved 2016-03-24.</li>
+               <li> Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li>
+            </ul>
+        
+        
+</body>
+</html>
```

Comment:
Align the code for all 3 files. There's a refactor option in VS Code itself.

Reply:
Press Shift+Alt+F in VS Code to indent your code.